examples/chip-tool/commands/clusters/ModelCommand.h

```diff
@@ -34,8 +34,8 @@ class ModelCommand : public CHIPCommand
 
     void AddArguments()
     {
-        AddArgument("node-id", 0, UINT64_MAX, &mNodeId);
-        AddArgument("endpoint-id", 0, UINT16_MAX, &mEndPointId);
+        AddArgument("node-id/group-id", 0, UINT64_MAX, &mNodeId);
+        AddArgument("endpoint-id/FabricIndex", 0, UINT16_MAX, &mEndPointId);
```

Comment:
Probably better to just do:
```suggestion
        AddArgument("endpoint-id-ignored-for-group-commands", 0, UINT16_MAX, &mEndPointId);
```
or something like that....

Reply:
We still need the fabricIndex for data retrieval within the GroupDataProvider since the groupId is unique within a fabric

Reply:
Done